index.html

```diff
@@ -1,90 +1,36 @@
-                    <p><b>Getting help:</b> For help with QIIME, see <a href="http://help.qiime.org">help.qiime.org</a>. For getting started on interacting with the command line, we recommend the <a href="http://software-carpentry.org/">Software Carpentry lessons and workshops</a>.</p>
-					<p><b>QIIME scripts:</b> <a href="http://qiime.org/scripts/index.html">The QIIME script documentation</a> will help you explore and learn QIIME's functionality.</p>
-					<h2> Code </h2>
-					<p>
-					QIIME is open source software. See <a href="https://github.com/biocore/qiime/blob/master/CONTRIBUTING.md">here</a> for information on how to contribute to QIIME.
-					</p>
-				</div>
-				<div>
-					<h2>Citing QIIME<a class="headerlink" href="#citing-qiime" title="Permalink to this headline"></a></h2>
-					<p>If you use QIIME for any published research, please include the following citation:</p>
-					<blockquote>
-						<div>
-							<p><strong>QIIME allows analysis of high-throughput community sequencing data</strong>
-							</p>
-							<p>J Gregory Caporaso, Justin Kuczynski, Jesse Stombaugh, Kyle Bittinger, Frederic D Bushman, Elizabeth K Costello, Noah Fierer, Antonio Gonzalez Pena, Julia K Goodrich, Jeffrey I Gordon, Gavin A Huttley, Scott T Kelley, Dan Knights, Jeremy E Koenig, Ruth E Ley, Catherine A Lozupone, Daniel McDonald, Brian D Muegge, Meg Pirrung, Jens Reeder, Joel R Sevinsky, Peter J Turnbaugh, William A Walters, Jeremy Widmann, Tanya Yatsunenko, Jesse Zaneveld and Rob Knight; Nature Methods, 2010; doi:10.1038/nmeth.f.303
-							</p>
-						</div>
-					</blockquote>
-					<p>QIIME wraps many other software packages, and these should be cited if they are used. For example, users of <tt>pick_de_novo.py</tt> are actually using QIIME, uclust, PyNAST, and FastTree, since QIIME is wrapping those applications. <b>Any time you're using tools that QIIME wraps, it is essential to cite those tools.</b> The Werner Lab website <a href="http://www.wernerlab.org/software/macqiime/citations">has a list of software, including citations, that is used in the various stages of QIIME</a>.
-					<p>You can find the <a class="reference external" href="http://www.nature.com/nmeth/journal/vaop/ncurrent/full/nmeth.f.303.html">QIIME paper here</a>, and the data presented in this paper can be found
-						<a class="reference external" href="ftp://ftp.microbio.me/qiime/QIIME_NM_2010.tgz">here</a>.
-					<br><br>
-					</p>
+	<header>
+		<img src="home_static/images/qiime2.png"/>
+	</header>
+	
+  	<div id="content">
+	<h1>QIIME has been succeeded by <a href="https://qiime2.org">QIIME 2</a>&nbsp; &nbsp;&nbsp;</h1>
```

Comment:
Should this be `QIIME&trade`? Or does that not matter?

Reply:
_Probably_ doesn't matter, but I added anyway. Thanks @wasade. 

Do you mind squashing and merging?

Reply:
![Screenshot 2022-12-19 at 11 43 56 AM](https://user-images.githubusercontent.com/192372/208497237-4b0034b9-bd6a-4b37-b09b-a930d20525af.png)